.github/workflows/publish_sdk.yaml

```diff
@@ -36,4 +36,27 @@ jobs:
         uses: pypa/gh-action-pypi-publish@master
         with:
           password: ${{ secrets.PYPI_API_TOKEN }}
-          packages_dir: packages/
\ No newline at end of file
+          packages_dir: packages/
+
+      - name: Send success notification
+        uses: rtCamp/action-slack-notify@v2
+        env:
+          SLACK_CHANNEL: criteo-api-sdk-generator
+          SLACK_COLOR: ${{ job.status }}
+          SLACK_MESSAGE: Python sdk publish succeeded. <https://github.com/criteo/criteo-api-python-sdk/actions/runs/${{github.run_id}}|Link to build>
+          SLACK_USERNAME: sdk-generation-bot
+          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
```

Comment:
I think we should add the secret into the repo configuration

Reply:
done